index.bs

```diff
@@ -4816,21 +4816,23 @@ let loopEnd;
 let playbackRate;
 
 // Variables for the node's playback parameters
-let start = 0, offset = 0; // Set by start()
-let stop = Infinity; // Set by stop(), or by start() with a supplied duration
+let start = 0, offset = 0, duration = Infinity; // Set by start()
+let stop = Infinity; // Set by stop()
+
 
 // Variables for tracking node's playback state
 let bufferTime = 0, started = false, enteredLoop = false;
+let bufferDuration = 0;
```

Comment:
Can we rename this?  Maybe `playbackDuration` or something? I find it confusing because there's `duration`, `buffer.duration`, and now `bufferDuration`.

Reply:
Sure, `playbackDuration` seems OK.

Reply:
Actually, I am now remembering that I called it `bufferDuration` because it is duration in terms of the buffer frame rate, _not_ the audible playback duration.

Reply:
So, what should it be?  I think that it's now really the audible playback duration, right?

Reply:
This PR is changing `duration` from a measurement in the audible timeline to a measurement across the buffer. i.e. the same units as `offset`.

Reply:
Now I'm thoroughly confused.  I thought if `start(when, offset,1)` were called and the playback rate is 1/10, the audio would be heard for 10 sec and not 1 sec.  (Assuming there's at least 1 sec of audio in the buffer, or we're looping.)

Reply:
Yes, for hearing the 10s.  playback would end when bufferDuration is 1.

Perhaps `bufferTimeElapsed`?

Reply:
@rtoy we agree on the meaning.  I'll let you modify the branch as heading out of touch for 2 weeks.

Reply:
@joeberkovitz I'll take over this PR.  Thanks for your work on this and see in a few weeks.

I think there's just a couple of minor things that need to be fixed:

* Rename `bufferDuration`
* Set `bufferTime` probably to just `offset`.

Reply:
Renamed `bufferDuration` and initialized `bufferTime` to `offset`.